lightning-config.h

```diff
@@ -0,0 +1,90 @@
+/* Generated by CCAN configurator */
+#ifndef CCAN_CONFIG_H
+#define CCAN_CONFIG_H
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE /* Always use GNU extensions. */
+#endif
+#define CCAN_COMPILER "aarch64-linux-android-clang"
```

Comment:
`aarch64-linux-android-clang` shouldn't be hardcoded but generated (alternative you could generate 4 one for each abi supported)